<?xml version="1.0" encoding="utf-8"?>	

<launch>

    <arg name="world_frame" default="odom"/>
    
    <arg name="respawn_value" default="false" /> <!-- boolean: true, false -->
      
	<node type="laser_proximity_checker" pkg="laser_proximity_checker" name="laser_proximity_checker" output="screen"  respawn="$(arg respawn_value)">
        <param name="laser_frame" value="/laser"/>
        <param name="robot_frame" value="/base_link"/>
        <param name="world_frame" value="$(arg world_frame)"/>

        <param name="max_size"       value="1000000"/>

        <!-- Deprecated functionality from NIFTi. -->
        <param name="publish2d"      value="false"/>
        <param name="relay_scans"    value="false"/>

        <!-- Subscribed -->
        <remap from="scan"               to="/scan_filtered"/>  <!-- DEPRECATED, not used. -->
        <remap from="scan_point_cloud"   to="/scan_point_cloud_color"/>
        <remap from="pointcloud_control" to="/pointcloud_control"/>
        
        <!-- Advertised -->
        <remap from="scan2d"                to="/scan2d"/>
        <remap from="scan_relay"            to="/scan_relay"/>
        <remap from="static_point_cloud"    to="/static_point_cloud"/>
        <remap from="dynamic_point_cloud"   to="/dynamic_point_cloud"/>
        <remap from="obst_point_cloud"      to="/obst_point_cloud"/>
        <remap from="laser_proximity_topic" to="/laser_proximity_topic"/>
        <remap from="closest_obst_point"    to="/closest_obst_point"/>
                
	</node>
</launch>





